test/FastTests/GenAi/GenAiBasics.cs

```diff
@@ -0,0 +1,532 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Runtime.InteropServices;
+using System.Threading;
+using System.Threading.Tasks;
+using Newtonsoft.Json;
+using Raven.Client;
+using Raven.Client.Documents.Operations.AI;
+using Raven.Client.Documents.Operations.ConnectionStrings;
+using Raven.Client.Documents.Operations.ETL;
+using Raven.Client.Documents.Operations.OngoingTasks;
+using Raven.Server.Documents;
+using Raven.Server.Documents.ETL.Providers.AI.GenAi;
+using Sparrow.Json;
+using Sparrow.Json.Parsing;
+using Tests.Infrastructure;
+using Xunit;
+using Xunit.Abstractions;
+
+namespace FastTests.GenAi;
+
+public class GenAiBasics(ITestOutputHelper output) : RavenTestBase(output)
+{
+    [RavenFact(RavenTestCategory.Ai)]
+    public void CanCreateGenAiTask()
+    {
+        using var store = GetDocumentStore();
+        store.Maintenance.Send(new PutConnectionStringOperation<AiConnectionString>(new AiConnectionString
```

Comment:
This will fail when Ollama isn't running - we need to first - skip it if it isn't running.
And we need to ensure that there _is_ an Ollama running, at least for the CI nightlies.

Reply:
done